scripts/generate_usb_vif/generate_vif.py

```diff
@@ -255,9 +252,34 @@ def parse_args():
                         help="device tree compatible to be parsed")
     parser.add_argument("--vif-out", required=True,
                         help="path to write VIF policies to")
-    parser.add_argument("--board", required=True, help="board name")
+    parser.add_argument("--vif-source-xml", required=True,
+                        help="XML file containing high level VIF specifications")
     return parser.parse_args()
 
 
+def main():
+    args = parse_args()
+    try:
+        with open(args.edt_pickle, 'rb') as f:
+            edt = pickle.load(f)
+    except edtlib.EDTError as err:
+        sys.exit(f"devicetree error: {err}")
+
+    xml_root = get_root()
+    source_xml_root = None
+    if args.vif_source_xml:
```

Comment:
If this arg is `required`, do you need to check this here?

Reply:
Yes it's not required, removing in new revision